Editor/Native/Android/SymbolsUpload.cs

```diff
@@ -184,7 +190,11 @@ private void UploadSymbols(string serverUrl, string symbolsToken, string symbols
                     {
                         EditorUtility.DisplayProgressBar("Backtrace symbols upload", "Symbols upload progress:", request.uploadProgress);
                     }
-                    var failure = request.isNetworkError || request.isHttpError;
+#if UNITY_2020_2_OR_NEWER
+                    var failure = request.result == UnityWebRequest.Result.ConnectionError || request.result == UnityWebRequest.Result.ProtocolError;
```

Comment:
Do we need to move the `var failure` outside of the #ifdefs like with `var disabledSymbols`? It looks like the #else condition here doesn't assign anything